OloEngine/src/OloEngine/Physics3D/JoltJobSystemAdapter.cpp

```
@@ -0,0 +1,85 @@
+    {
+        // Jobs are freed automatically by the free list destructor
+    }
+
+    i32 JoltJobSystemAdapter::GetMaxConcurrency() const
+    {
+        // Return the number of worker threads in the scheduler
+        // FScheduler tracks workers; we add +1 for the calling thread that can also execute jobs during WaitForJobs
+        return static_cast<i32>(LowLevelTasks::FScheduler::Get().GetNumWorkers()) + 1;
+    }
+
+    JPH::JobSystem::JobHandle JoltJobSystemAdapter::CreateJob(const char* inName, JPH::ColorArg inColor,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)